python/plugins/processing/gui/AlgorithmDialog.py

```diff
@@ -165,9 +165,11 @@ def getParameterValues(self):
                     parameters[param.name()] = value
                     if param.isDestination():
                         context = dataobjects.createContext()
+                        #TODO: could we ignore below check if parameter is optional?
```

Comment:
Yes - looks like a bug in isSupportedOutputValue. Can you drop this commit from the PR and I'll fix the c++ method?

Reply:
Fixed in https://github.com/qgis/QGIS/pull/9554

Reply:
Cool!.
 I will update this PR

Reply:
Ok, #9554 is merged now